.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+// Coloque su configuración en este archivo para sobrescribir la configuración predeterminada y de usuario.
```

Comment:
If we want to share some configuration, i recommend using an .editorconfig file (http://editorconfig.org/). It works out of the box in various editors, and in other ones (brackets, atom, vscode) a plugin is needed.